### PR TITLE
Dev cleanup

### PR DESCRIPTION
**Pull request by cyrusnaficy**

### CODE REVIEW

**Comment by cyrusnaficy** [comment]
@mbardoe Some formatting changes from our meeting yesterday

**Comment by cyrusnaficy** [comment]
> @mbardoe Some formatting changes from our meeting yesterday

Before merging plz wait for new feat/intake to be merged to dev. I think its an in progress PR now.

**Review by gmsong06 — APPROVED** [review]

**Pull request merged**
